libindy/android.test.sh

```diff
@@ -73,6 +71,9 @@ execute_on_device(){
     adb -e push \
     "${LIBZMQ_LIB_DIR}/libzmq.so" "/data/local/tmp/libzmq.so"
 
+    adb -e push \
```

Comment:
is this line required because we drop unit tests from the CI scope?

Reply:
Yep. Libindy is not compiled into any more.